release_notes/47-release-notes.adoc

```diff
@@ -0,0 +1,381 @@
+include::snippets/technology-preview.adoc[]
+
+In {product-title-short} 4.7, you can now bring vulnerability findings directly into the {rh-rhdh} and other platforms based on the upstream Backstage project by using the Backstage Plugin.
+You can incorporate the {product-title-short} plugin into your developer environment and review key vulnerabilities for your applications within the platform. This provides information that you need before you can take the necessary remediation actions.
+
+////
+For more information, see xref:../configuration/configuring-and-integrating-the-rhacs-plugin-with-red-hat-developer-hub.adoc#viewing-security-information-in-red-hat-developer-hub_configure-and-integrate-rhacs-plugin[Viewing security information in Red Hat Developer Hub].
+////
+
+//ROX-24052
+//tech preview
+[id="generate-sboms-from-the-scanned-container-images_{context}"]
+=== Generate SBOMs from the scanned container images
+
+:FeatureName: Generation of SBOMs from the scanned container images
+include::snippets/technology-preview.adoc[]
+
+With {product-title-short} 4.7, you can now generate a Software Bill of Materials (SBOM) from the scanned container images. These SBOMs give you a detailed overview of all software components, dependencies, and libraries within your application.
+{product-title-short} creates SBOMs of the Analyzed type and complies with the Software Package Data Exchange (SPDX) 2.3 specification.
+
+For more information about the SBOMs of the Analyzed type, see
+link:https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf[Types of SBOM Documents] (America's Cyber Defense Agency documentation).
+
+For more information about how to use {product-title-short} to generate SBOMs, see xref:../operating/manage-vulnerabilities/scanner-generate-sbom.adoc#scanner-generate-sbom[Generating SBOMs from scanned images].
+
+[NOTE]
+====
+Generating SBOMs from images analyzed through secured clusters by using the delegated scanning feature is currently not supported. It is anticipated that an upcoming release will support this feature.
+====
+
+//ROX-17535
+[id="automatic-certificate-rotation-for-secured-clusters_{context}"]
+=== Automatic certificate rotation for secured clusters
+
+{product-title-short} 4.7 introduces automatic certificate rotation for the secured cluster components, which ensures secure communication between Sensor, Admission Controller, and Collector within the cluster. Sensor also uses these certificates for a secure certificate-based architecture (mTLS) connection with Central.
+
+With this release, you can now use the Cluster Registration Secret (CRS) to bootstrap a secured cluster and register it with Central. Unlike previous methods, CRS separates the bootstrap credentials from the operational certificates so that the cluster continues to work even if the bootstrap credentials are revoked. You can generate the CRS by using the `roxctl` CLI commands and revoke it if required.
```

Comment:
This section starts with cert rotation, then mentions CRS, then goes back to cert rotation. I think CRS should be mentioned separately, as it's a different feature.

Reply:
If we need to break out the CRS into a different section, we can add a link to the docs. (Won't work until they're merged).

For more information about CRS, see xref:../../installing/installing_ocp/init-bundle-ocp.adoc#init-bundle-ocp[Generating and applying an init bundle or cluster registration secret for RHACS on Red Hat OpenShift] and xref:../../installing/installing_other/init-bundle-other.adoc#init-bundle-other[Generating and applying an init bundle or cluster registration secret for RHACS on other platforms].